### ISSUE TITLE
Add org.apache.royale

### ISSUE THREAD

**Issue by yishayw** [jvm-repo-rebuild/reproducible-central#117]
Hi,

Version 0.9.10 of org.apache.royale is reproducible. What steps do I need to take to add it to your list?

Thanks

**Comment by hboutemy** [comment]
i can work on it: is it https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/apache/royale/compiler/README.md ?
if not, please provide at least a pointer to the Git repo

**Comment by yishayw** [comment]
Yes. The project actually consists of 3 different repos the compiler you mentioned, the application framework at apache/royale-asjs: Apache Royale ASJS (github.com)<https://github.com/apache/royale-asjs> and the so called 'typedefs' at apache/royale-typedefs: Apache Royale TypeDefs (github.com)<https://github.com/apache/royale-typedefs>

They all get released together and are tested for binary reproducibility.

Out of the three only the compiler is coded in Java and it is used in other projects as well, so that might be the best fit out of the three for this list if you had to choose.

Thanks!


[https://opengraph.githubassets.com/8b4643e85cbac0f98bcd5362e6f93afdca2d6e3932d17b74dd06875984f68c35/apache/royale-asjs]<https://github.com/apache/royale-asjs>
GitHub - apache/royale-asjs: Apache Royale ASJS<https://github.com/apache/royale-asjs>
Apache Royale ASJS. Contribute to apache/royale-asjs development by creating an account on GitHub.
github.com

________________________________

**Comment by hboutemy** [comment]
as you can see, I tried to rebuild the compiler and found differences between my local build and the reference build from Maven Central: https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/apache/royale/compiler/royale-maven-plugin-0.9.10.diffoscope

perhaps you can help me improve the buildspec if some issues are cause by inaccurate build instructions: https://github.com/jvm-repo-rebuild/reproducible-central/blob/master/content/org/apache/royale/compiler/royale-compiler-0.9.10.buildspec

to rebuild as Reproducible Central does, run
`./rebuild.sh content/org/apache/royale/compiler/royale-compiler-0.9.10.buildspec`

**Comment by yishayw** [comment]
I'll discuss this with the team and see if we can figure this out.

**Comment by yishayw** [comment]
It turns out we do not check src or doc jars for reproducibility, just the binary ones.
